CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 ## In Development
 
 
+## v0.30.0
+* Pin st2 version to `v3.3dev` as a new latest development version (#129)
+* Migrate from `py2` `Ubuntu Xenial` to `py3` `Ubuntu Bionic` as a base StackStorm OS (StackStorm/st2-dockerfiles#16, #129)
+* Switch from MongoDB `3.4` to `4.0` for the mongodb-ha Helm chart (#129)
+* Update `etcd-operator` 3rd party chart from `0.10.0` to latest `0.10.3` (#129)
+* Update `rabbitmq-ha` 3rd party chart from `1.36.4` to `1.44.1` (#129)
+* Update `mongodb-replicaset` 3rd party chart from `3.9.6` to `3.14.0` (#129)
```

Comment:
Spot https://github.com/helm/charts/pull/21630#issuecomment-628285592 bug along the way.